tensorflow/core/util/ctc/ctc_decoder.h

```diff
@@ -98,8 +101,8 @@ class CTCGreedyDecoder : public CTCDecoder {
         auto row = input[t].row(b);
         int max_class_ix;
         (*scores)(b, 0) += -row.maxCoeff(&max_class_ix);
-        if (max_class_ix != blank_index_ &&
-            !(merge_repeated_ && max_class_ix == prev_class_ix)) {
+        if (max_class_ix != Decoder::blank_index_ &&
```

Comment:
ditto; undo

Reply:
same as in https://github.com/tensorflow/tensorflow/pull/31164#discussion_r312368512